testdata/sample_config.yaml

```diff
@@ -12,7 +12,6 @@ system: .metadata.annotations."opslevel.com/system"
 tier: .metadata.annotations."opslevel.com/tier"
 repositories:
   - '{"name": "My Cool Repo", "directory": "", "repo": .metadata.annotations.repo} | if .repo then . else empty end'
-  - .metadata.annotations.repo
```

Comment:
Think about this "sample config" for a second - we're telling the customer to make a repository using `.metadata.annotations.repo` and to AGAIN make the same repository, except this time with even less detail (no `name` and `directory`) - why? This just result in a conflict 100% of the time.

Reply:
I think the idea here is to show off different approaches of how a repository could be managed. It looks like 3 approaches toward getting any repo and also shows that we support multiple repos. 
In fact, this is actually a good thing for us since our code should be able to handle duplicate entries.

I'm still reading into these updates but am leaning towards keeping this config as is

Reply:
@davidbloss OK that's fine -- but can we make it at least use a different variable? So for example

```
  - '{"name": "My Cool Repo", "directory": "", "repo": .metadata.annotations.repository} | if .repo then . else empty end'
  - .metadata.annotations.repo
```

If both lines are pointing towards `metadata.annotations.repo` that will 100% cause a collision.

Reply:
@davidbloss I added this change, this should make more sense.